src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Verify that the `Person` component is correctly exported from './components/Person/Person'. Ensure the file path is accurate and the component is available for import.